HACKING

```diff
@@ -341,6 +341,22 @@ Example::
     {
         ...
 
+Translation
+^^^^^^^^^^^
+
+Strings which are visible to geany users need to be translated and thus
+should be enclosed in ``_( )`` to have them being recognized by the GNU
+translation tools. Also spaces right after the first ``"`` of a string
+shall not be used as they will be cut of by the tools. This is a bad example::
+
+    label = gtk_label_new (_("First sentence of a bad case example."
```

Comment:
No space between the function name and the open parenthesis in Geany code style.